### DISCUSSION TITLE
Suggested lists disappear after using one

### DISCUSSION THREAD

**Discussion by senden9** [community/community#8644] [Other Feature Feedback, Questions, & Ideas]
HI!

After using of the suggested lists the others disappear. See before first usage:
![before first usage](https://user-images.githubusercontent.com/905857/145530750-f4bb2f2a-056f-4886-8d71-5720a552d374.png)
After adding one repository to the list "My stack" I see only this one list on other repositories:
![after first usage](https://user-images.githubusercontent.com/905857/145530768-93386685-df70-4ed8-8c41-e5e785e838d2.png)
This is quite a surprising behaviour for me. It is also a bit sad because the proposals were good proposals :D

**Comment by atc0005** [comment]
I also found this surprising. It is probably worth keeping the suggestions until at least X lists have been created by the user.

**Comment by gcanahuirev** [comment]
I only selected 2, the third one disappeared :disappointed_relieved:

**Comment by JasonWeinzierl** [comment]
For those of you wanting to re-create these three lists after they've disappeared:

- 🔮 Future ideas
- 🚀 My stack
- ✨ Inspiration

Emojis are named "crystal_ball", "rocket", and "sparkles".

**Reply by KANGOD** [reply]
Just searched expecting this!
